preview/scripts/edit-description.js

```diff
@@ -33,7 +41,8 @@ module.exports = async ({inputs, github, context}) => {
         pull_number: context.issue.number,
     });
 
-    const body_message = MESSAGE_TEMPLATE.replace("{docs-pr-index-url}", RTD_URL);
+
+    const body_message = MESSAGE_TEMPLATE.replace("{docs-pr-index-url}", RTD_URLS.join("\r\n"));
```

Comment:
It would be good if we can prepend a `* ` on each line, so it appears as a bullet list.